### PR TITLE
Upgrade to the latest version of pyee

### PR DESCRIPTION
**Pull request by forslund**
Updates the pyee module to the latest version. A change is prepared for mycroft-core updating this as well.

The code changes the now deprecated pyee.EventEmitter to the equivalent pyee.BaseEventEmitter

### CODE REVIEW

**Comment by forslund** [comment]
Ping @clusterfudge

**Comment by forslund** [comment]
Ping @clusterfudge ?

File: adapt/engine.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def __init__(self, tokenizer=None, trie=None):
                 example EnglishTokenizer()
             trie(Trie): tree of matches to Entites
         """
-        pyee.EventEmitter.__init__(self)
+        pyee.BaseEventEmitter.__init__(self)
```
Comment (clusterfudge) [inline adapt/engine.py:48]: Is this change necessary? I didn't have to make the change as part of my PR #106 , and all tests continue to pass.

Comment (forslund) [reply adapt/engine.py:48]: It's not necessary but using EventEmitter gives a depreciation warning.

Comment (clusterfudge) [reply adapt/engine.py:48]: ACK. Happy to take your PR, but it needs to bump version in setup.py.

Comment (forslund) [reply adapt/engine.py:48]: Pushed version bump
